staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,60 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called ".import-restrictions". If this file is not found, parent directories will be recursively searched.
+  - NOTE: Adding ".import-restrictions" files does not add them to CI runs. In order for them to be added, it needs to be explicitly added to `arguments.InputDirs`, like so:
```

Comment:
seems better to also quote ".import-restrictions" like so `.import-restrictions`

Reply:
isn't --input-dirs also a flag on the command line?

Reply:
Yes it is. Nice catch 👍